### DISCUSSION TITLE
Atomiswave kof 11

### DISCUSSION THREAD

**Discussion by Juniorsouza123** [flyinghead/flycast#1686] [General]
Good evening, I would like to know if anyone has noticed that kof xi atomiswave on the scene loading screen has the loading cut off. However, in previous versions of flycast it was similar to the arcade version. I think they will change something in the code. Thank you
![0](https://github.com/user-attachments/assets/6d5be6f3-b7a3-47dc-ba15-29452ecb41f6)
![0](https://github.com/user-attachments/assets/47d786ca-1aaa-4ba7-905d-05844cddb695)

**Comment by flyinghead** [comment]
Flycast version(s)?

**Reply by Juniorsouza123** [reply]
in version a50164a flycast the stage kof 11 loading screen is cutting off the loading before time.

**Comment by Juniorsouza123** [comment]


**Reply by flyinghead** [reply]
please answer my question

**Reply by Juniorsouza123** [reply]
Ok

**Reply by flyinghead** [reply]
Thank you.
I created issue #1724 to track this case.

**Reply by flyinghead** [reply]
Fixed on the `dev` branch

**Reply by Juniorsouza123** [reply]
thanks  you flying head incredible work in preserving arcades
